spec/services/media_storage/azure_adapter_spec.rb

```diff
@@ -8,7 +8,7 @@
   let(:private_container) { 'secret-magic-container' }
   let(:opts) do
     {
-      url: 'https://test-uploads.zooniverse.org/container_name',
+      domain_prefix: 'https://test-uploads.zooniverse.org/container_name',
```

Comment:
i don't think `domain_prefix` matches `url_prefix`
```suggestion
      url_prefix: 'https://test-uploads.zooniverse.org/container_name',
```